src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -60,6 +61,8 @@ export interface AmalgamationFilingIF {
     incorporationAgreement?: {
       agreementType: string
     }
+    resolutions?: ResolutionIF[]
```

Comment:
It is optional, or could you just save `[]` if there aren't any resolutions? Which do you think is better?

Reply:
save `[]` approach is better

Reply:
I agree. Remove the optional character, initialize it as `[]` in the store state, and make sure saving/filing an empty list works.